source/agora/node/BlockStorage.d

```diff
@@ -572,7 +572,14 @@ public class BlockStorage : IBlockStorage
     {
         try
         {
-            this.readBlockAtPosition(block, this.height_idx.findBlockPosition(height));
+            const block_pos = this.height_idx.findBlockPosition(height);
+            if (height != 0 && block_pos == 0)
```

Comment:
Can `findBlockPosition` fail for `height = 0`?

Reply:
No, `height = 0` means `pos = 0`